aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -489,6 +490,17 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"1K", "2K", "4K", "8K", "16K"}, false),
 				Description:  "Gateway ethernet interface RX queue size. Supported for AWS related clouds only.",
 			},
+			"private_mode_lb_vpc_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "Private Mode Controller load balancer VPC ID. Required when private mode is enabled for the Controller.",
+				ConflictsWith: []string{"allocate_new_eip"},
+			},
+			"private_mode_subnet_zone": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Private Mode subnet availability zone.",
+			},
 			"availability_domain": {
```

Comment:
For the new attributes in spoke, transit and vpc, can they be updated?

Reply:
Good catch, they should be force new.